README.md

```diff
@@ -49,7 +49,7 @@ or installable packages must be specified there. They should not be moved to
 `install.bash`as`tue-env` has many controls in place to parse the YAML file.
 
 ### Naming conventions
-Name of the target must start with `ros-` only if it will be compiled/installed by ROS or it is installed by reading the dependencies from a catkin `package.xml`
+Name of the target must start with `ros-` only if it will be compiled/installed by ROS.
```

Comment:
If it contains a catkin package?

Reply:
If the package is a catkin package or a ros system package then it must start with `ros-`. Otherwise no.

I have implemented a fail safe there. For any ros target installation, its parent package will be first searched with `ros-` if not found, then it will be looked for without. If neither are found then an error will be thrown.

To test what I mean just install `ros-audio_capture`. You will understand what's happening.

Reply:
(all system ros packages also are catkin packages)

Reply:
Basically what I mean is any target that its `install.yaml` as:
```yaml
- type: ros
  source: ...
```
must have its target name starting with `ros-`